src/schema/rules/sidecars/emg.yaml

```diff
@@ -73,14 +73,6 @@ EMGRecommended:
     HardwareFilters: recommended
     RecordingDuration: recommended
 
-EMGEpoched:
-  selectors:
-    - datatype == "emg"
-    - suffix == "emg"
-    - sidecar.RecordingType == "epoched"
-  fields:
-    EpochLength: required
```

Comment:
realized after our conversation @effigies that you'd already done the generalization I was asking about, in `src/schema/rules/sidecars/electrophys.yaml` so I just tacked EMG on to there and removed our EMG-specific rule.  It's a slight behavior change (we were making EpochLength required, the general rule makes it recommended) but IMO that's OK

Reply:
I would suggest keeping it required for EMG. It was recommended for the others because we could not promote it to required for backwards compatibility reasons. That said, we could duplicate the rule in `electrophys.yaml` with required/recommended variants so that they're still consolidated.

Reply:
ah yeah, I forgot that the rationale for "recommended" was backwards compatibility. I've now done as you suggest, in [04b2638](https://github.com/bids-standard/bids-specification/pull/1998/commits/04b2638d8b098641aaefed5251b0e2248069809a)